component/src/main/java/org/wso2/extension/siddhi/io/http/sink/HttpRequestSink.java

```diff
@@ -360,10 +367,23 @@
                         optional = true,
                         defaultValue = "null",
                         dynamic = true),
+                //added
```

Comment:
I guess you have added this for your reference. Please remove such comments before committing the code.

Reply:
removed from the code